static/elements/chromedash-featurelist.js

```diff
@@ -369,16 +351,24 @@ class ChromedashFeaturelist extends LitElement {
   render() {
     return html`
       <link rel="stylesheet" href="/static/css/elements/chromedash-featurelist.css">
-
-      ${this.filtered.map((feature) => html`
-        <div class="item">
-          <div ?hidden="${this._computeMilestoneHidden(feature, this.features, this.filtered)}"
-               class="milestone-marker">${this._computeMilestoneString(feature.browsers.chrome.status.milestone_str)}</div>
-          <chromedash-feature id="id-${feature.id}" tabindex="0"
-              @feature-toggled="${this._onFeatureToggled}"
-              .feature="${feature}" ?whitelisted="${this.whitelisted}"></chromedash-feature>
-        </div>
-        `)}
+      <style>
+        .item {width: 100%}
+      </style>
+
+      <lit-virtualizer
+        .scrollTarget=${window}
+        .items=${this.filtered}
+        @rangechange=${this._onScrollList}
+        .renderItem=${(feature) => html`
+          <div class="item">
+            <div ?hidden="${this._computeMilestoneHidden.call(this, feature, this.features, this.filtered)}"
+                 class="milestone-marker">${this._computeMilestoneString.call(this, feature.browsers.chrome.status.milestone_str)}</div>
+            <chromedash-feature id="id-${feature.id}" tabindex="0"
+                 @feature-toggled="${this._onFeatureToggled.bind(this)}"
```

Comment:
Rather than bind a new closure each render, write `_onFeatureToggled` as a class property with an arrow function

```js
constructor() {
  this._onFeatureToggled = (e) => { ... };
}
```

and

```
@feature-toggled="${this._onFeatureToggled}"
```

Reply:
Done